src/hotspot/share/jfr/dcmd/jfrDcmds.cpp

```diff
@@ -72,7 +68,28 @@ static bool is_disabled(outputStream* output) {
 
 static bool invalid_state(outputStream* out, TRAPS) {
```

Comment:
The changes look okay to me except that "invalid_state" isn't a clear name for  function that tries to ensure that the jdk.jfr module is loaded. Up to you, but I think it would be clear if it were renamed load_jfr that returns disabled when disabled or the jdk.jfr module cannot be loaded.